src/core/attributeformmodelbase.cpp

```diff
@@ -606,7 +606,14 @@ void AttributeFormModelBase::updateVisibilityAndConstraints( int fieldIndex )
       {
         allConstraintsHardValid = false;
         if ( mHasTabs && item->parent() )
-          item->parent()->setData( false, AttributeFormModel::ConstraintHardValid );
+        {
+          QStandardItem *parentItem = item->parent();
+          while ( parentItem->parent() )
+          {
+            parentItem = parentItem->parent();
+          }
+          parentItem->setData( false, AttributeFormModel::ConstraintHardValid );
```

Comment:
Do we want to take that into the loop?

Reply:
@m-kuhn , I don't think so no, we just want the root tab to get flag, not all subgroups along the way.

Reply:
I thought that could be interesting too, even if it's not visualized at the moment?